### PR TITLE
issue-1-create-an-entry | Angel Leung

### PR DESCRIPTION
**Pull request by angelsleung**

### CODE REVIEW

**Comment by angelsleung** [comment]
issue-1-create-an-entry

File: js/data.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
 /* exported data */
+function beforeUnload(event) {
+  var entriesJSON = JSON.stringify(data.entries);
```
Comment (codingcodymiller) [inline js/data.js]: You should be storing the entire data model in the `data` variable, not just the `entries` property

File: js/data.js [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
 var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
   nextEntryId: 1
 };
+
+var previousEntriesJSON = localStorage.getItem('entries-local-storage');
+if (previousEntriesJSON !== null) {
+  data.entries = JSON.parse(previousEntriesJSON);
```
Comment (codingcodymiller) [inline js/data.js]: Sam here, you will be reassigning the parsed value here to `data`, not the `entries` property.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! Please see my notes

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳 🥳

**Pull request merged**